### ISSUE TITLE
undefined method '[]' for nil:NilClass when no defaults specified

### ISSUE THREAD

**Issue by desmondmonster** [renz45/mandrill_mailer#86]
perhaps similar to https://github.com/renz45/mandrill_mailer/issues/77

With the following code: 
```
class MyMailer < MandrillMailer::TemplateMailer

  def csv opts = {}
    new_lead_count = opts[:new_lead_count]

    yesterday = Date.yesterday.strftime('%Y-%m-%d')

    mandrill_mail(
      template: 'new-activity-report',
      subject: "New Activity summary for #{yesterday}",
      to: 'desmond@crevalle.io',
      from: 'messages@client.com',
      vars: {
        'DATE' => yesterday,
        'NEW_LEAD_COUNT' => new_lead_count
      }
     )
  end
end
```

I get the following error:

```
NoMethodError: undefined method `[]' for nil:NilClass
/Users/desmond/.gem/ruby/2.1.2/gems/mandrill_mailer-0.6.0/lib/mandrill_mailer/core_mailer.rb:346:in `merge_default_merge_vars'
/Users/desmond/.gem/ruby/2.1.2/gems/mandrill_mailer-0.6.0/lib/mandrill_mailer/core_mailer.rb:329:in `mandrill_args'
/Users/desmond/.gem/ruby/2.1.2/gems/mandrill_mailer-0.6.0/lib/mandrill_mailer/template_mailer.rb:162:in `mandrill_mail'
/Users/desmond/code/my_app/app/mailers/my_mailer.rb:29:in `csv'
/Users/desmond/.gem/ruby/2.1.2/gems/mandrill_mailer-0.6.0/lib/mandrill_mailer/core_mailer.rb:288:in `call'
/Users/desmond/.gem/ruby/2.1.2/gems/mandrill_mailer-0.6.0/lib/mandrill_mailer/core_mailer.rb:288:in `method_missing'
/Users/desmond/code/my_app/app/events/create_csv_report.rb:21:in `strategy'
/Users/desmond/.gem/ruby/2.1.2/bundler/gems/happenings-05d2bdcc2e15/lib/happenings/event.rb:20:in `block in run!'
/Users/desmond/.gem/ruby/2.1.2/bundler/gems/happenings-05d2bdcc2e15/lib/happenings/event.rb:99:in `time'
/Users/desmond/.gem/ruby/2.1.2/bundler/gems/happenings-05d2bdcc2e15/lib/happenings/event.rb:19:in `run!'
```
Adding `default from: 'info@client.com'` fixes it.  It seems I need at least one `default` option at the class level, or the `@defaults` var never gets set in `MandrillMailer::CoreMailer`.  Maybe specify in the docs that calling `.default` is required?

**Comment by renz45** [comment]
I just released version 1.0.0, which should fix this issue. :)

**Comment by desmondmonster** [comment]
🎉
